packages/FormElement/src/FormElement.js

```diff
@@ -87,14 +87,18 @@ function FormElement(props) {
 
   function renderFooter() {
     if (hasError) {
-      return React.cloneElement(extractedChildren["FormElement.Error"], {
-        ariaErrorId,
-      });
+      return (
+        <div aria-live="assertive">
```

Comment:
Interestingly here we are rendering the `div` with the `aria-live` in the same render cycle as its content and this works fine on chrome. In our accessibility course we were advised that the container div would have to be rendered pre-contents, so will be good to test this . Will also be a good idea to test this approach on other accessibility platforms (jaws, navx?).